packages/lib/modules/tokens/TokenInput/TokenInput.tsx

```diff
@@ -188,6 +188,11 @@ function TokenInputFooter({
             getPriceImpactLabel(priceImpactProps?.priceImpact)}
         </Text>
       )}
+      {!isBalancesLoading && isMounted && hasError && (
+        <Text color="font.error" fontSize="sm">
+          {getValidationError(token)}
+        </Text>
+      )}
```

Comment:
moved the position of error msg from right next to wallet balance to the middle 

before:

<img width="652" height="172" alt="image" src="https://github.com/user-attachments/assets/99677608-56a5-443a-a191-1e2a989420e9" />

after: 

<img width="673" height="167" alt="Image" src="https://github.com/user-attachments/assets/9c7db8c5-fb70-49a2-8ba1-43574c0666c5" />